services/file-storage-and-transfer/brown-storage.md

```diff
@@ -3,9 +3,233 @@ title: Brown Storage Options
 icon: database
 ---
 
-Several services at Brown allow you to share and store files. These different storage services each have their own relative benefits and features. You can explore the differences with our comparison tool using the link below.
+Several services at Brown allow you to share and store files. These different 
+storage services each have their own relative benefits and features. You can 
+explore the differences with our comparison tool using the link below.
 
-<a href="/storage" class="button is-link">Compare Storage Options</a>
-<a href="https://brown.atlassian.net/servicedesk/customer/portal/16/group/55/create/218" class="button is-link">Request Storage</a>
-<a href="https://brown.atlassian.net/servicedesk/customer/portal/16/group/55/create/219" class="button is-link">Request Quota Increase</a>
-<a href="https://brown.atlassian.net/servicedesk/customer/portal/16/group/55/create/217" class="button is-link">Storage Help</a>
+<details>
+    <summary>Campus File Storage Non-replicated / Research Services</summary>
+    <p>
+        File Services for Research provides Brown University research departments 
+        with a location in which files can be stored, backed up, and shared with members 
+        of the Brown community using Brown ID’s and groups. Space is allocated to each 
+        research lab or PI with an ITHelp request , security groups are required to define 
+        access to the data. The data is protected locally via snapshots but doesn’t have 
+        geo-redundancy (secondary copy).
```

Comment:
Is there a decent place to link to a definition of geo-redundancy and why this might be an issue for data?